src/Cubes/Cubes.jl

```diff
@@ -484,7 +484,7 @@ function Base.getindex(a::YAXArray, args::DD.Dimension...; kwargs...)
                 if v isa UnitRange{Int}
                     v = Date(first(v))..Date(last(v),12,31)
                 end
-                d2[:Ti] = v
```

Comment:
This code path will not be hit anymore, because time is not a Ti anymore. 
We might want to get rid of this anyway.

Reply:
I will test 😄.

Reply:
I agree we wanted to remove this anyway and since we will make a breaking release lets get rid of this

Reply:
done already on the latest commit.